tests/unit/test_connection.py

```diff
@@ -42,13 +41,24 @@ def __init__(self, responses):
         {'request-id': 2},
         {'request-id': 3},
     ]
-    con._get_sll = mock.MagicMock()
+    minimal_facades = [{'name':'Pinger','versions':[1]}]
+    con = None
     try:
-        with mock.patch('websockets.connect', base.AsyncMock(return_value=ws)):
-            await con.open()
+        with \
```

Comment:
I think this kind of blocks are slightly more readable if written as
```python
with (
    mock.patch('websockets.connect', base.AsyncMock(return_value=ws)),
    mock.patch(...),
):
    con = await Connection.connect('0.1.2.3:999')
```
So with avoiding back slashes and making more clear where the block actually starts.

Reply:
Unfortunately that's not an option for `with` arguments (it doesn't accept a tuple in that way).

Reply:
[Looks like](https://docs.python.org/3/reference/compound_stmts.html#the-with-statement) the multiple context expression version of `with` is 3.1 and greater.

Reply:
As Roger pointed out to me on IRC, even in Python 3, the parenthetical, no-backslash syntax is in fact rejected.